jenkins/osa_onmetal.functions.groovy

```diff
@@ -370,6 +370,18 @@ def stop_api_uptime_tests() {
     """
 }
 
+def parse_results() {
+
+    String host_ip = get_onmetal_ip()
+
+    //Pull persistent, during, api, smoke results from onmetal to ES vm
+    sh """
+    git clone https://github.com/osic/elastic-benchmark
+    sudo pip install -e elastic-benchmark
+    elastic-upgrade -u output/output.txt -d output/during_output.txt -p output/persistent_resource.txt -b subunit/before_upgrade -a subunit/after_upgrade
```

Comment:
That is where Anthony's script parses and places the formatted persistence test output.